app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumCategoriesActivityTest.kt

```diff
@@ -0,0 +1,100 @@
+package com.github.h3lp3rs.h3lp.forum
+
+import android.view.View
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.h3lp3rs.h3lp.R
+import com.github.h3lp3rs.h3lp.USER_TEST_ID
+import com.github.h3lp3rs.h3lp.signin.SignInActivity
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ForumCategoriesActivityTest {
+
+    @get:Rule
+    val testRule = ActivityScenarioRule(
+        ForumCategoriesActivity::class.java
+    )
+
+    @Before
+    fun setup() {
+        SignInActivity.globalContext = ApplicationProvider.getApplicationContext()
+        SignInActivity.userUid = USER_TEST_ID
+        Intents.init()
+    }
+
+    @After
+    fun clean() {
+        Intents.release()
+    }
+
+    private fun clickingOnButtonWorksAndSendsIntent(ActivityName: Class<*>?, id: Matcher<View>) {
+        Espresso.onView(id).perform(ViewActions.scrollTo(),ViewActions.click())
+        Intents.intended(
+            Matchers.allOf(
+                IntentMatchers.hasComponent(ActivityName!!.name)
```

Comment:
Similarly here (and for all the other tests below), importing ViewMatchers, Espresso and ViewActions makes the tests much easier to read and make more sense.